featurebyte/query_graph/graph.py

```diff
@@ -280,22 +280,31 @@ def load(self, graph: QueryGraphModel) -> Tuple["QueryGraph", Dict[str, str]]:
             node_name_map[node.name] = node_global.name
         return self, node_name_map
 
-    def extract_operation_structure(self, node: Node, **kwargs: Any) -> OperationStructure:
+    def extract_operation_structure(
+        self,
+        node: Node,
+        keep_all_source_columns: bool,
```

Comment:
Would it make sense/be possible to set the default value here to be True?

Reply:
Yes, that's my initial fix. In the end, feel like making it explicit is better so that it forces us to be aware this parameter. I don't have strong opinion here.

Reply:
If there are only limited cases where it should be set as False and it's risky to do so, having this default to True makes sense to me.

Reply:
Will update this in the coming fix.